src/main/scala/com/libertyglobal/odh/kafkatool/Main.scala

```diff
@@ -267,12 +269,34 @@ object Main extends StrictLogging {
         updateCommand(kafka, config, opts.update.alterIfNeeded.getOrElse(false), opts.update.dryRun.getOrElse(false))
       case Seq(c) if c == opts.listSuperfluousTopics =>
         listSuperfluousTopicsCommand(kafka, config)
+      case Seq(c) if c == opts.listAcls =>
+        listAcls(kafka)
+      case Seq(c) if c == opts.updateAcls =>
+        applyAcls(kafka, config, opts.update.dryRun.getOrElse(false))
+
       case _ =>
         opts.printHelp()
         sys.exit(1)
     }
   }
 
+  private def listAcls(kafka: AdminClient): Unit = {
+    ACLManager.list(kafka).foreach(acl => logger.info(acl.toString))
+  }
+
+  private def applyAcls(kafka: AdminClient, config: KafkaToolConfig, dryRun: Boolean = false): Unit = {
+    val acls = config.getAcls()
+    if (dryRun) {
+      ACLManager.list(kafka).foreach(acl => logger.info("To-be deleted ACL " + acl.toString))
```

Comment:
I'd say 
```scala
for (acl <- ACLManager.list(kafka)) {
  logger.info(s"To-be deleted ACL ${acl}")
}
```
looks more concise

Reply:
`.foreach` is slightly less readable than `for` loop. Sometimes it's easier to "end the sentence" with `.foreach` than "start the sentence" with `for` but usually it's the case when you have a lot of `map`s, `flatmap`s and other operations in a long chain.